### PR TITLE
add callback func

### PR DESCRIPTION
**Pull request by Chyroc**

### CODE REVIEW

**Comment by Chyroc** [comment]
https://github.com/Chyroc/WechatSogou/issues/72

**Comment by Chyroc** [comment]
https://github.com/Chyroc/WechatSogou/issues/71

**Comment by Chyroc** [comment]
https://github.com/Chyroc/WechatSogou/issues/62

File: test/__init__.py [hunk 1]
```diff
@@ -1 +1,12 @@
 # -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals, print_function
```
Comment (h3z) [inline test/__init__.py]: review

**Pull request merged**